app/stac_api/management/commands/dummy_asset_upload.py

```diff
@@ -0,0 +1,117 @@
+import logging
+import os
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+from stac_api.models import Asset
+from stac_api.models import AssetUpload
+from stac_api.models import BaseAssetUpload
+from stac_api.s3_multipart_upload import MultipartUpload
+from stac_api.utils import AVAILABLE_S3_BUCKETS
+from stac_api.utils import CommandHandler
+from stac_api.utils import get_sha256_multihash
+
+logger = logging.getLogger(__name__)
+
+
+class DummyAssetUploadHandler(CommandHandler):
+
+    def __init__(self, *args, **kwargs):
+
+        s3_bucket = kwargs.pop('s3_bucket', AVAILABLE_S3_BUCKETS.legacy)
+        print(
+            f"connect MultipartUploader to s3 bucket "
+            f"{settings.AWS_SETTINGS[s3_bucket.name]['S3_BUCKET_NAME']}"
+        )
+        super().__init__(*args, **kwargs)
+        self.uploader = MultipartUpload(s3_bucket)
+
+    def start(self):
+        self.print_success(f"Starting upload for {self.options['asset_id']}")
+        asset = Asset.objects.filter(name=self.options['asset_id']).first()
+
+        key = f"{asset.item.collection.name}/{asset.item.name}/{asset.name}"
```

Comment:
FYI there's a function for that: `def get_asset_path(item, asset_name):` in `utils.py`